### DISCUSSION TITLE
How to calculate total mass of CO2 injected in paraview?

### DISCUSSION THREAD

**Discussion by wdmhouston** [GEOS-DEV/GEOS#1450] [Q&A]
Hi,
     Can you tell me which variable in paraview can be used to calculate total CO2 mass in reservoir?
![image](https://user-images.githubusercontent.com/19333848/120260184-fd73c500-c25a-11eb-8f26-4ff7e53aacc5.png)

Thanks.

**Comment by wdmhouston** [comment]
Never mind,   I manually calculated in pavaview(Volume*ReferencePoro*MassDensity) and the total CO2 mass reported in model is close to the total the CO2  mass specified (injection rate*time).

**Reply by beckyggbbh** [reply]
Hi @wdmhouston , can you please explain me what parameters in paraview did you use?
